server/channels/api4/job_local.go

```diff
@@ -9,6 +9,7 @@ func (api *API) InitJobLocal() {
 	api.BaseRoutes.Jobs.Handle("", api.APILocal(getJobs)).Methods(http.MethodGet)
 	api.BaseRoutes.Jobs.Handle("", api.APILocal(createJob)).Methods(http.MethodPost)
 	api.BaseRoutes.Jobs.Handle("/{job_id:[A-Za-z0-9]+}", api.APILocal(getJob)).Methods(http.MethodGet)
+	api.BaseRoutes.Jobs.Handle("/{job_id:[A-Za-z0-9]+}/download", api.APILocal(downloadJob)).Methods(http.MethodGet)
```

Comment:
Was this just an oversight from the time when we added the REST endpoint, but forgot to add a local equivalent?

Reply:
Looks like it.